internal/logger/logger.go

```diff
@@ -170,10 +170,10 @@ func (f *loggerFactory) writer(level string) io.Writer {
 
 	switch level {
 	case "NOTICE":
-		return daemonize.StatusWriter
+		return f.createJsonOrTextWriter(level, daemonize.StatusWriter)
 	case "ERROR":
-		return os.Stderr
+		return f.createJsonOrTextWriter(level, os.Stderr)
```

Comment:
what does it mean by wrapping os.stderror or os.stdout with json or text writer. Does it change anything?

Reply:
Yes, after that log will be written via JSON/TEXT writer which will add time stamp with the log-message.
https://github.com/GoogleCloudPlatform/gcsfuse/blob/d97394ecca7150e4ead3070a529d182cf25b66b0/internal/logger/writer.go#L73

Reply:
will it still log to stdout or to file?

Reply:
This will log to std-out file only. But the message goes via text/json writer, which add extra information like time-stamp and log-level to the logging message.

I have updated the description before and after the change. There will not be any change in log-output type. It will still write to std-out only.